static/app/views/settings/earlyFeatures/settingsForm.tsx

```diff
@@ -8,19 +8,20 @@ import JsonForm from 'sentry/components/forms/jsonForm';
 import type {JsonFormObject} from 'sentry/components/forms/types';
 import LoadingIndicator from 'sentry/components/loadingIndicator';
 import {t} from 'sentry/locale';
-import type {Organization, OrganizationAuthProvider, Scope} from 'sentry/types';
+import type {OrganizationAuthProvider, Scope} from 'sentry/types';
 import {useApiQuery} from 'sentry/utils/queryClient';
-import withOrganization from 'sentry/utils/withOrganization';
+import useOrganization from 'sentry/utils/useOrganization';
```

Comment:
looks like some `useOrganization` snuck into the PR too

Reply:
ya, i did them both when they were around. i can re-title it

Reply:
the target is the projects

there might also be one or two usePageFilters()

Reply:
oh, more of the txn stuff used that. so stay tuned for that last pr